tests/e2e/test_e2e_template.py

```diff
@@ -526,12 +527,13 @@ def test_make_train_multiple_experiments(
             assert job in dumped_jobs, f"dumped jobs: {dumped_jobs}"
 
         run("make kill-train-all", detect_new_jobs=False)
-        jobs_left = run(
-            f'bash -c "neuro ps | grep {MK_TRAIN_JOB}"',
-            assert_exit_code=False,
-            detect_new_jobs=False,
-        )
-        assert not jobs_left
+
+        # TODO: neuro-ps + grep is blocked by neuro issue:
+        #  https://github.com/neuromation/platform-client-python/issues/1189
+        # hyper_ls = f"bash -c 'neuro ps | grep {mk_train_job('-hyper-')}'"
+        # running = run(hyper_ls, assert_exit_code=False).splitlines()
+        # assert not running, f"expect no hyper-jobs: {running}"
```

Comment:
this change and onwards -- currently not failing but potentially incorrect until we fix `neuro`